social-network/src/App.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import './App.css';
+import Card from './Card/Card';
+import MissedArticles from './data/missed-articles.json';
+import YourArticles from './data/your-articles.json';
+
+class App extends Component {
+  render() {
+    /**
+    * Loop through YourArticles data and build out cards
+    * @param {object} item - The article object in the array
+    */
+    const listItems = YourArticles.map((item, ind) => {
+      const { link, image, description, title, author, postedDate, minutesToRead } = item;
```

Comment:
I wouldn't destructure the properties here, but rather pass the whole {item} as a prop to Card component.   This way the component is responsible for its own internals